2023/CloudExpertsGroup/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "rijwanansari",
+      "SatyaKarki"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Cloud Experts Group",
+    "Country": "Philippines",
+    "Url": "https://www.facebook.com/groups/CloudExpertsGroup",
+    "SessionizeApiId": "1qyb8j6r",
+    "SessionizeEventName": "https://sessionize.com/global-azure-philippines-2023/",
+    "Logo": {
+      "Description": "Global Azure Bootcamp Philippines 2023 is a community event organized by the Cloud Experts Group and .NET Girl jointly. Join us on 13 May 2023 for a FREE Online event to inspire, learn and deep dive into Azure and Cloud Computing.",
+      "File": "_template.png"
```

Comment:
This file name needs to indicate your file in your folder that corresponds to your logo. That would be CloudExpertsGroup.png. Our logic that puts your info on the web site cannot make assumptions! ;)